academic-requirements/src/components/Semester.tsx

```diff
@@ -19,6 +17,7 @@ const style: CSSProperties = {
   fontSize: "1rem",
   lineHeight: "normal",
   float: "left",
+  whiteSpace: "pre",
```

Comment:
This line of CSS seems to causing (or is related to) the horizontal scrollbar appearing when hovering a course over a semester. However, removing it causes issues text wrapping for the semester credit counter.

Reply:
It looks like I got the scroll bar fixed, don't really know why it fixed it but 🤷